src/main/scala/no/ndla/articleapi/repository/ArticleRepository.scala

```diff
@@ -40,7 +40,7 @@ trait ArticleRepository {
       article.copy(id=Some(articleId), revision=Some(startRevision))
```

Comment:
Gjelder egentlig to linjer ovenfor, men vil ikke den nye flyten kreve at du ikke regenerer id'en? Eller er det håndtert i `updateAndReturnGeneratedKey()`?

Reply:
Jepp, det stemmer. Var ganske usikker på hvordan jeg skulle håndtere dette siden vi gjerne vil at article-api skal fungere som vanlig helt til versjoneringen er ferdig. Når draft-api fungerer og article-api brukes kun for publisering tenker jeg å gå over å slette alt av funksjonalitet som ikke lengre er nødvendig. Denne er en av dem som vil bli borte. Andre kandidater er public endepunkter for å opprette og oppdatere artikler.

Når en ID allokeres fra draft-api legges det inn en tom rad i databasen. Når en artikkel publiseres vil denne oppdateres.